docassemble/assemblylinewizard/setup.py

```diff
@@ -0,0 +1,44 @@
+import os

```

Comment:
I don't think this `setup.py` is needed?

Reply:
Good catch! I probably uploaded to the wrong folder and forgot about it.